README.md

```diff
@@ -47,3 +51,9 @@ cat <<EOF > server.json
 EOF
 pm2 start server.json
 ```
+
+## TESTING
```

Comment:
Well, I think we better make a new PR dealing with it. How do you think @thebearjew ?